proto/engine/v1/BUILD.bazel

```diff
@@ -111,8 +113,10 @@ go_library(
 ssz_proto_files(
     name = "ssz_proto_files",
     srcs = [
+        "electra.proto",
+        "fulu.proto",
         "execution_engine.proto",
-	"electra.proto",
+        "execution_engine_eip7594.proto",
```

Comment:
why are we add in this file ?

Reply:
Fixed in 619d9ba90df4ab909bfd83256287641517955fb7.